templates/plug_html_template.tmpl

```diff
@@ -0,0 +1,23 @@
+<html>
+	<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">
+                <center>
+		<h1 style="color: white;">Title Goes Here!</h1>
+		<h2 style="color: white;">Subtitle goes here!</h2>
+		<p style="color: white";>
+                        Informative text goes here
+		</p>
+		<div>
+		<p style="color: white";>
+                        Whatever form/image/other thing you want can go here

```

Comment:
This is where you'd want to put some of the template-specific syntax, an example of how to do this can be found at https://github.com/gin-gonic/gin#html-rendering